packages/paste-core/components/side-panel/stories/index.stories.tsx

```diff
@@ -0,0 +1,223 @@
+                margin="space40"
+                width="size50"
+                height="size90"
+              >
+                I am a box with a red border and I have fixed height and width
+              </Box>
+            </Box>
+          </SidePanelPushContentWrapper>
+          <SidePanel label="the important side panel">
+            <SidePanelHeader>
+              <Heading as="h5" variant="heading30" marginBottom="space0">
+                Heading
+              </Heading>
+              <SidePanelHeaderActions>
+                <Button variant="secondary_icon" size="reset" onClick={() => {}}>
+                  <MoreIcon decorative={false} title="open menu" size="sizeIcon50" />
+                </Button>
+              </SidePanelHeaderActions>
+            </SidePanelHeader>
+            <SidePanelBody>
+              <Box
+                width="100%"
+                height="100%"
+                backgroundColor="colorBackgroundDecorative10Weakest"
+                borderBottomLeftRadius="borderRadius50"
+                borderBottomRightRadius="borderRadius50"
+              />
+            </SidePanelBody>
+          </SidePanel>
+        </SidePanelContainer>
+      </SidebarPushContentWrapper>
+    </Box>
+  );
+};
+
+ContentDemo.parameters = {
+  padding: false,
+};
+
+export const Composed = (): React.ReactNode => {
+  const [isOpen, setIsOpen] = React.useState(true);
+
+  const topbarSkipLinkID = useUID();
+  const mainContentSkipLinkID = useUID();
+
+  return (
+    <Box>
+      {/* Sidebar can be placed anywhere - position fixed */}
+      <SidebarWithContent topbarSkipLinkID={topbarSkipLinkID} mainContentSkipLinkID={mainContentSkipLinkID} />
+      <SidebarPushContentWrapper collapsed={false} variant="compact">
+        <Topbar id={topbarSkipLinkID}> </Topbar>
+        <SidePanelContainer isOpen={isOpen} setIsOpen={setIsOpen}>
+          <SidePanelPushContentWrapper>
+            <MessagingInsightsContent mainContentSkipLinkID={mainContentSkipLinkID} />
+          </SidePanelPushContentWrapper>
+          {/* Side Panel can be placed anywhere - position fixed */}
+          <SidePanelWithContent />
+        </SidePanelContainer>
+      </SidebarPushContentWrapper>
+    </Box>
+  );
+};
+
+Composed.parameters = {
+  padding: false,
+};
+
+export const Customized = (): React.ReactNode => {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const label = useUID();
+  const theme = useTheme();
+
+  return (
+    <CustomizationProvider
+      theme={theme}
+      elements={{
+        SPPW: { margin: "space50" },
+        SPPCW: { paddingRight: "space150" },
+        SP: { top: "30px" },
+        ANIMATED_SP_WRAPPER: { backgroundColor: "colorBackgroundPrimaryWeaker", borderRadius: "borderRadius80" },
+        INNER_SP: { paddingBottom: "space40" },
+        SPH: { borderBottomWidth: "borderWidth40" },
+        SPHA: { justifyContent: "flex-start" },
+        SPB: { width: "75%" },
```

Comment:
If there isn't a reason for using acronyms here, I think it would be preferable to see full component names.